UT4MasterServer/Controllers/SessionController.cs

```diff
@@ -379,7 +379,7 @@ public async Task<IActionResult> Verify()
 
 		var obj = new JObject()
 		{
-			{ "token", user.AccessToken },
+			{ "access_token", user.AccessToken },
```

Comment:
having `token` here is actually not a mistake. that's what epic's servers respond with, so it should stay as is.